.travis.yml

```diff
@@ -41,10 +41,12 @@ install:
     else
       go get;
     fi
+  - [ $DEP_MANAGEMENT_TOOL == "DEP" ] || go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.16.0
```

Comment:
I am wondering what you want to achieve here. Do you mean if `$DEP_MANAGEMENT_TOOL == "DEP"` and then `go get ...`?

Reply:
I'm not sure if we need to run a linter in *every* version of go, or just the latest version of Go.  A lot of the linting tools are newer and only work on the latest version of Go.

I think it makes a lot of sense to run unit tests or integration tests on every version of Go, but unsure what advantage we get out of limiting linters to just those supported on 1.9.

Reply:
Oh I misunderstood your question!

I'm trying to install golangci-lint only if modules are enabled, which is only if we're using modules and not dep.

Reply:
in this case, we can make if else condition(there are some example inside the .travis.yaml file)